packages/cube/lib/sort.js

```diff
@@ -0,0 +1,15 @@
+import { Readable, Duplex } from 'stream'
+import { sort, compareOn, createStore } from 'external-merge-sort'
+import rdf from '@zazuko/env-node'
+
+async function write(chunk, filename) {
+  await rdf.toFile(Readable.from(chunk), filename)
+  return rdf.fromFile(filename)
+}
```

Comment:
Not exported. Unused?

Reply:
used a few lines below:
`const store = createStore(write, '.nt')`

This is the function that spills to temporary files on disk when the number of items to sort exceeds a threshold

Reply:
Ah, ok. Code coloring misled me